src/views/CashlinkManage.vue

```diff
@@ -0,0 +1,348 @@
+<template>
+    <div class="container pad-bottom" v-if="retrievedCashlink">
+        <SmallPage class="cashlink-receive">
+            <transition name="transition-fade">
+                <StatusScreen v-if="!isTxSent" state="loading" :title="title" :status="status" lightBlue/>
+            </transition>
+
+            <PageBody>
+                <transition name="transition-fade">
+                    <div v-if="!isManagementRequest && isTxSent" class="nq-green cashlink-status"><CheckmarkSmallIcon/>Cashlink created</div>
+                </transition>
+                <button class="nq-button-s close" @click="close">Done</button>
+                <div class="cashlink-and-url">
+                    <Account :displayAsCashlink="true" layout="column" :class="{'sending': !isTxSent, 'show-loader': !isManagementRequest}"/>
+                    <Copyable :text="link">
+                        <div class="cashlink-url">{{link}}</div>
+                    </Copyable>
+                </div>
+            </PageBody>
+            <PageFooter v-if="nativeShareAvailable">
+                <button class="nq-button copy" :class="copied ? 'green' : 'light-blue'" @click="copy">
+                    <span v-if="copied"><CheckmarkSmallIcon /> Copied</span>
+                    <span v-else>Copy</span>
+                </button>
+                <button class="nq-button share-mobile" @click="share">
+                    Share
+                </button>
+            </PageFooter>
+            <PageFooter v-else>
+                <button class="nq-button copy" :class="copied ? 'green' : 'light-blue'" @click="copy">
+                    <span v-if="copied"><CheckmarkSmallIcon /> Copied</span>
+                    <span v-else>Copy</span>
+                </button>
```

Comment:
Instead of duplicating the Copy button between the two layouts, how about applying the condition to the Share button and the group of social buttons instead?